src/components/List/List.tsx

```diff
@@ -233,11 +144,11 @@ export class List extends React.Component {
                 break;
             }
             case 'PageDown': {
-                this.handleKeyMove(event, pageSize);
+                this.handleKeyMove(event, pageSize!);
```

Comment:
Are we confident there is a value here?

Reply:
Page size can be `undefined` in case of initial state. It is bad idea to change logic in case of moving code from js to ts